lib/raster3d/index.c

```diff
@@ -59,7 +59,7 @@ static int Rast3d_readIndex(RASTER3D_Map * map)
 	    return 0;
 	}
 
-	Rast3d_rle_decode(tmp2, tmp, map->indexLongNbytes * map->nTiles, 1,
```

Comment:
Change the type of tmp2, tmp at the beginning of `Rast3d_readIndex()`?

Reply:
```c
void Rast3d_long_decode(unsigned char *, long *, int, int);
```
is used a few lines later. Either `Rast3d_long_decode()` must be redefined, or reverse the cast.

Reply:
OK. Seems to be an inconsistency in the raster3d lib.